### ISSUE TITLE
Roshan spawns while game paused

### ISSUE THREAD

**Issue by mgAnto** [jeffhill/Dota2#1219]
### Description

During the recent games we have been experiencing a lot of package loss, around 4%. Sometimes a reconnect fixes it but then it comes back again.

But this game had the weirdest bug. We were waiting for roshan, had a couple of disconnects. The game was paused at 36:43, and while it was paused we had Ancient Apparition in the rosh pit giving us vision. Roshan spawned at that time while the game was paused. I couldnt check if the Rosh timer was ticking while paused. Screenshots attached showing both instances of pit without roshan and with roshan, at the same time while game was paused.

I am pretty sure this unintended and should be fixed ASAP.

Thank you Jeff

### Example Match ID (and possibly Timestamp)

6616366143 36:43

### Screenshots

![6616366143_1](https://user-images.githubusercontent.com/107458715/173507281-6ac6425a-e2fd-46d7-a5b9-6e53f265c02a.png)
![6616366143_2](https://user-images.githubusercontent.com/107458715/173507294-4c319841-549d-4e95-b4fb-8357daffa0cb.png)

**Comment by DrQuint** [comment]
The game can advance a single server tick if someone levels up an ability during the pause. This can not be done anymore with regular abilities, but it can with stats (the ones underneath talents). Can you double check that no one leveled their stats at that exact moment in the match?

**Comment by pontuslaestadius** [comment]
@DrQuint I don't think that is the case for this bug, I went through the heroes and didn't spot it.

https://user-images.githubusercontent.com/31597765/173626287-251ca38f-3eb8-4a87-85ff-aa9691aba7e8.mp4
